### PR TITLE
Flesh out the proposals README with more information about the process.

### PR DESCRIPTION
**Pull request by TrevorFSmith**
These changes are intended to clarify the CG process for taking feature ideas through to standardization, including some sense of the work involved and how to get support from the CG leadership.

### CODE REVIEW

File: ProposalsIssueTemplate.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Proposals Issue Template
+
+TBD
+
+This doc will hold guidelines for new proposal Issues as well as a cut-and-paste template to improve the quality of Issues. 
+
+## Guidelines for filing a good issue
+To ensure that your proposal receives the attention it deserves, there are some basic guidelines you should follow:
+
+  - Do a quick [search](https://github.com/immersive-web/proposals/issues?q=is%3Aissue) for your proposal before posting, and if there's an open conversation please join it instead.
```
Comment (cwilso) [inline ProposalsIssueTemplate.md]: Should they search the WebXR spec issues, too?

File: ProposalsIssueTemplate.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - Give your issue a descriptive title. "VR idea" will likely get ignored and closed. "Better VR video" is slightly more descriptive, but still very ambiguous. "Proposal for handling protected video content" is a far more informative title.
+  - Your proposal should suggest a direction for addressing the issue. Stating "I think WebXR content should be faster." isn't particularly helpful, while "Technology X is now available, which can improve render times. It could potentially be exposed to WebXR like this..." is a much more effective starting point for a conversation. However, please be open to alternative approaches and healthy discussion. (See [Code of Conduct](#code-of-conduct).)
+  - Describing real world use cases for your proposal is always plus! What would it enable that couldn't be done before, or improve over what can be done today?
```
Comment (cwilso) [inline ProposalsIssueTemplate.md]: I think this should be a requirement, not just a plus.  :)

File: README.md [hunk 3]
```diff
@@ -1,42 +1,80 @@
...
+An immersive web feature goes through several steps to move from idea to standard:
+
+- File a Proposals Issue
+- Find Consensus for the Idea
+- Recruit Feature Advocates
+- Create a Feature Repo
+- Write an Explainer
+- Find Consensus for Feature Details
+- Recruit Implementers
+- Pitch a Working Group
```
Comment (cwilso) [inline README.md]: Pitch *to* a Working Group?

File: README.md [hunk 4]
```diff
@@ -1,42 +1,80 @@
...
 
 ## I have a proposal! How do I get started?
-To get the ball rolling with a proposal, file an issue on this repo. That issue will host discussion for the proposal until such time that it's either given its own repo or closed as will not implement. Note that the issue does not have to have sustained discussion to remain open: It may take a while for some proposals to attract the attention necessary to move forward, and that's OK!
 
-## Do I have to have a spec/explainer/proof of concept before posting an proposal?
-No! Certainly if you have any of those available it's great to share them, but it's absolutely not necessary. At minimum you should have a clear goal in mind and a proposed direction for how to address it.
+Great! If the paragraphs above haven't scared you off and you're up for the social work of standardization then you should file a GitHub Issue in the [proposals repo](https://github.com/immersive-web/proposals/issues) using the [Proposals Issue Template](ProposalsIssueTemplate.md).
```
Comment (cwilso) [inline README.md]: I would re-pitch that they should do a search first.  Or at least say "if the above hasn't scared you off, and you've already done a search..."

File: README.md [hunk 5]
```diff
@@ -1,42 +1,80 @@
...
+
+This repository is for *open discussion of ideas* for making the Immersive Web better and we foster by an environment where all discussion is respectful. Every person participating in the community group is expected to follow the [Code of Conduct](https://github.com/immersive-web/community-resources/blob/master/CODE_OF_CONDUCT.md).
+
+## How does a feature idea become a standard?
+
+An immersive web feature goes through several steps to move from idea to standard:
+
+- File a Proposals Issue
+- Find Consensus for the Idea
+- Recruit Feature Advocates
```
Comment (cwilso) [inline README.md]: It's hard to tell if "feature advocate" is an explicit title/role, and if it's the same as the "feature champion" you mention below.  Do you just mean "members interested in this issue"?

File: README.md [hunk 6]
```diff
@@ -1,42 +1,80 @@
...
-  - Your proposal should suggest a direction for addressing the issue. Stating "I think WebXR content should be faster." isn't particularly helpful, while "Technology X is now available, which can improve render times. It could potentially be exposed to WebXR like this..." is a much more effective starting point for a conversation. However, please be open to alternative approaches and healthy discussion. (See [Code of Conduct](#code-of-conduct).)
-  - Describing real world use cases for your proposal is always plus! What would it enable that couldn't be done before, or improve over what can be done today?
+Common case: People agree that the feature is interesting but you don't gain much momentum and nobody steps up to work on the feature. The Issue is eventually closed by the CG chair but it stays in the archive as a reference. Occasionally a closed proposal Issue will be re-animated in the future because someone steps up to be the feature champion.
```
Comment (cwilso) [inline README.md]: I would change the "issue is eventually closed by the CG chair" line to be more descriptive of why this might happen (so members wouldn't think that you'd just start closing out their issues to show off your godlike powers :).  Maybe something like "If an issue discussion goes completely stagnant, the CG chair may close it, but will leave it in the archive in case interest is re-animated and someone steps up to be the feature champion."

File: README.md [hunk 7]
```diff
@@ -1,42 +1,80 @@
...
-  - If you have an issue with a specific spec, open an issue on that spec first.
+Terrific! You did the hard work of getting enough momentum in the group to figure out what you're making and how it would work. The next step is to recruit one or two people from other organizations who want to join you as feature advocates. This isn't a formal W3C community group role; it simply means that each of you agrees to do the work of nailing down a feature's details and working with the rest of the group to find consensus on its details.
 
-So what happens if you get it wrong? Nothing bad! We'll point you towards a more appropriate venue if necessary and, when possible, help migrate the issue for you. And please don't assume that a request to continue discussion of your issue elsewhere is a criticism, we genuinely want to ensure that your idea is reaching the right audience!
+Once two or three people from separate organizations have volunteered to be feature advocates then you should reach out to the CG chair and ask that a feature repo be created.
```
Comment (cwilso) [inline README.md]: Is "you need two or three people from separate orgs signed up to be feature advocates first" really going to be the bar for creating a repo?  Because you'll end up pushing a lot of people into personal repos for feature design up front.

Maybe I'm just reacting to the difference between "you need at least one or two other people saying this is interesting to discuss" (which is the WICG bar) and "two or three others have volunteered to be feature advocates" which sounds like they're 100% bought in to the feature (and feels like a high bar).

File: README.md [hunk 8]
```diff
@@ -1,42 +1,80 @@
...
+
+The other (perhaps more important) goal of the repo is to attract the attention of implementers and topic experts and then find consensus for the feature details.
+
+Feature advocates (mentioned in the paragraphs above) are on the hook to do both technical and social work. It can take **a lot** of effort to gather enough feedback from a wide enough range of people that your feature will work for the entire immersive web.
+
+You start by writing an explainer document that lays out the context for the feature, captures the main discussion points from the proposals Issue, and runs through a few possible shapes that the feature's API or other technical artifacts could take. The goal of this document is to give readers the information they need to decide whether they want to work on the feature.
+
+Again, if you get stuck on this step (either in the writing or in the social aspects) then reach out to a community group chair. 
+
+## Our feature repo has all of the details need to implement a feature. Now what?
```
Comment (cwilso) [inline README.md]: need => needed

File: README.md [hunk 9]
```diff
@@ -1,42 +1,80 @@
...
 
-So what happens if you get it wrong? Nothing bad! We'll point you towards a more appropriate venue if necessary and, when possible, help migrate the issue for you. And please don't assume that a request to continue discussion of your issue elsewhere is a criticism, we genuinely want to ensure that your idea is reaching the right audience!
+Once two or three people from separate organizations have volunteered to be feature advocates then you should reach out to the CG chair and ask that a feature repo be created.
 
-## Code of Conduct
-**TL;DR: Treat other participants with respect or get banned.**
+## The CG chair made a new feature repo. Now what?
+
+One goal of the repo is to work through the nitty-gritty details of a feature with enough detail that it's clearly possible to implement and it's attractive to the relevant Working Group.
```
Comment (cwilso) [inline README.md]: "nitty-gritty" might be less understandable to a non-English-as-a-first-language participant.  "Specific details"?

File: README.md [hunk 10]
```diff
@@ -1,42 +1,80 @@
...
+
+## Our feature repo has all of the details need to implement a feature. Now what?
+
+Excellent! You've talked to enough of the community and worked together as feature advocates to nail down pretty much everything that one would need to know in order to implement a feature.
+
+If you've been prototyping the feature during the process of nailing down the design, you're probably in good shape. If not, you'll need to find someone to write code to prove out the design so that you can make the case that the feature is definitely ready to use in production.
+
+Ideally, you'll have prototypes that work on different platforms so it's clear that the feature is not a solution for just one vendor.
+
+During this process you should reach out to a CG chair and start the work of finding or creating a W3C working group that is a good fit for the feature.
```
Comment (cwilso) [inline README.md]: Here you say "a CG chair", elsewhere you say "the CG chair".  I'd just make it consistent.
